docs/src/prototypes/chatMessages/ChatMessageWithPopover/ChatWithPopover.tsx

```diff
@@ -1,124 +1,177 @@
+              '& a': {
+                color: siteVariables.colors.brand[600],
+              },
+            }),
+          },
+          Menu: {
+            root: {
+              background: '#fff',
+              transition: 'opacity 0.2s',
               position: 'absolute',
-              opacity: 0,
-              zIndex: -1,
-            },
 
-            '&.focused .smile-emoji': {
-              position: 'initial',
-              zIndex: 'initial',
-              opacity: 1,
-            },
+              '& a:focus': {
+                textDecoration: 'none',
+                color: 'inherit',
+              },
+              '& a': {
+                color: 'inherit',
+              },
+
+              '& .smile-emoji': {
+                position: 'absolute',
+                opacity: 0,
+                zIndex: -1,
+              },
+
+              '&.focused .smile-emoji': {
+                position: 'initial',
+                zIndex: 'initial',
+                opacity: 1,
+              },
 
-            '&:hover .smile-emoji': {
-              position: 'initial',
-              zIndex: 'initial',
-              opacity: 1,
+              '&:hover .smile-emoji': {
+                position: 'initial',
+                zIndex: 'initial',
+                opacity: 1,
+              },
             },
           },
         },
-      },
-    }}
-  >
-    <Chat
-      items={[
-        {
-          key: 'a',
-          message: {
-            content: (
-              <Chat.Message
-                actionMenu={<Popover />}
-                author="Jane Doe"
-                content={{
-                  content: (
-                    <div>
-                      <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
-                    </div>
-                  ),
-                }}
-                timestamp="Yesterday, 10:15 PM"
-              />
-            ),
-          },
-          gutter: { content: <Avatar {...janeAvatar} /> },
-        },
-        {
-          key: 'b',
-          message: {
-            content: (
-              <Chat.Message
-                actionMenu={<Popover />}
-                author="Jane Doe"
-                content={{
-                  content: (
-                    <div>
-                      <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
-                    </div>
-                  ),
-                }}
-                timestamp="Yesterday, 10:15 PM"
-              />
-            ),
+      }}
+    >
+      <Chat
+        items={[
+          {
+            key: 'a',
+            message: {
+              content: (
+                <Chat.Message
+                  actionMenu={
+                    <Popover setFixedMode={setFixedModeCM1} setOpen={val => setOpenCM1(val)} />
+                  }
+                  onMouseEnter={() => setOpenCM1(true)}
+                  onMouseLeave={() => !fixedModeCM1 && setOpenCM1(false)}
```

Comment:
On mouse leave we will close the action menu, unless it is in fixed mode (the submenu is opened).